core/src/com/unciv/ui/mapeditor/LoadScenarioScreen.kt

```diff
@@ -20,7 +20,9 @@ class LoadScenarioScreen(previousMap: TileMap?): PickerScreen(){
     init {
         rightSideButton.setText("Load scenario".tr())
         rightSideButton.onClick {
-            UncivGame.Current.setScreen(MapEditorScreen(MapSaver.loadScenario(chosenScenario)).apply { scenarioName = chosenScenario })
+            val mapEditorScreen = MapEditorScreen(MapSaver.loadScenario(chosenScenario))
+            mapEditorScreen.apply { mapName = chosenScenario }
```

Comment:
If you already have a MapEditorScreen constructor that accepts a Scenario, then setting the mapName can be done in the constructor =)

Reply:
Done! Will be fixed in the next commit.